src/book.py

```diff
@@ -158,7 +157,16 @@ def compare_books(self, other: LimitOrderBook) -> bool:
             other_dll = other._bids[level]
             if not dll.is_equal(other_dll):
                 logging.error(f"FAIL: Bids Price level {level} mismatching orders")
-            failed = True
```

Comment:
This was intended incorrectly previously, so the presence of any bid leads to this function returns orderbook mismatch.